ui/app/src/components/BulkCancelPackageDialog/BulkCancelPackageDialogContainer.tsx

```diff
@@ -50,26 +49,30 @@ export function BulkCancelPackageDialogContainer(props: BulkCancelPackageDialogC
 	const { formatMessage } = useIntl();
 
 	const handleSubmit = () => {
-		dispatch(updateBulkCancelPackageDialog({ isSubmitting: true }));
+		dialogId && dispatch(updateDialogState({ id: dialogId, props: { isSubmitting: true } }));
```

Comment:
Don't like the idea of needing to share the `dialogId` in here, unless we somehow did it as a pattern. But all enhanced dialogs have in context their update submit or has pending functions so don't see the need. Check/update all.